packages/tasks/src/tasks/image-classification/inference.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
+ *
+ * Using src/scripts/inference-codegen
+ */
+
+/**
+ * Inputs for Image Classification inference
+ */
+export interface ImageClassificationInput {
+	/**
+	 * On or several image files to classify
+	 */
+	inputs: unknown;
+	/**
+	 * Additional inference parameters
+	 */
+	parameters?: ImageClassificationParameters;
+	[property: string]: unknown;
+}
+
+/**
+ * Additional inference parameters
+ *
+ * Additional inference parameters for Image Classification
+ */
+export interface ImageClassificationParameters {
+	/**
+	 * When specified, limits the output to the top K most probable classes.
+	 */
+	topK?: number;
+	[property: string]: unknown;
+}
+
+/**
+ * Outputs of inference for the Image Classification task
+ */
+export interface ImageClassificationOutput {
```

Comment:
I think the output is a list of these

Reply:
Yes - it should be outputed as such by the code generation tool as the spec specifies correctly this should be an array

Same issue for `FeatureExtraction` output not being generated - I'll look into it